.env.example

```diff
@@ -15,7 +15,7 @@ VUE_APP_SOCIETIES_ONLINE_HOME_URL="https://dev.bcregistry.ca/societies/"
 VUE_APP_AUTH_API_URL="https://auth-api-dev.apps.silver.devops.gov.bc.ca"
 VUE_APP_AUTH_API_VERSION="/api/v1"
 VUE_APP_LEGAL_API_URL="https://legal-api-dev.apps.silver.devops.gov.bc.ca"
-VUE_APP_LEGAL_API_VERSION="/api/v1"
+VUE_APP_LEGAL_API_VERSION="/api/v2"
```

Comment:
The Legal API "fetch business" call doesn't work on V1, but I need to test the "create business" call on V2.

Reply:
Tested! `BusinessServices.createBusiness()` works fine on V2 endpoint.

Reply:
I added a "todo" item to my ticket to remember to update 1Password.